examples/bindgen_toolchain/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+load("@bazel_skylib//rules:native_binary.bzl", "native_binary")
+load("@rules_cc//cc:defs.bzl", "cc_import")
+load("@rules_rust_bindgen//:defs.bzl", "rust_bindgen_toolchain")
+
+native_binary(
+    name = "clang",
+    src = "@llvm_toolchain_llvm//:bin/clang",
+)
+
+cc_import(
+    name = "libclang",
+    shared_library = "@llvm_toolchain_llvm//:libclang",
+)
+
+rust_bindgen_toolchain(
+    name = "my_bindgen_toolchain",
+    bindgen = "@rules_rust_bindgen//3rdparty:bindgen",
+    clang = ":clang",
```

Comment:
Why the `native_binary` for this attribute?

Reply:
`@llvm_toolchain_llvm//:bin/clang` is a filegroup, not an executable Label, so it needs to be "converted" with a genrule with executable=True, or a native_binary.